test/refda/test_exec.c

```diff
@@ -55,6 +59,15 @@ static refda_agent_t agent;
 /// Sequence of executions
 static cace_ari_list_t exec_log;
 
+static void test_reporting_edd_one_int(refda_edd_prod_ctx_t *ctx)
+{
+    cace_ari_t result = CACE_ARI_INIT_UNDEFINED;
+    atomic_int val    = atomic_load(&edd_backing_value);
```

Comment:
Sorry for the churn but I think the type of val here should be plain `int`

Reply:
OK, changed `val` to a plain `int`.